runtime-decompiler/src/main/java/org/jrd/frontend/frame/main/BytecodeDecompilerView.java

```diff
@@ -414,7 +414,7 @@ public void actionPerformed(ActionEvent actionEvent) {
             @Override
             public void componentResized(ComponentEvent e) {
                 if (splitPaneFirstResize) {
-                    splitPane.setDividerLocation(0.5);
```

Comment:
@commit message: where is some pack?

Same, same.

Reply:
moved to different commit and fixed